sample/src/test/java/app/cash/paparazzi/sample/ComposeA11yTest.kt

```diff
@@ -56,7 +57,13 @@ class ComposeA11yTest {
           )
           Column {
             Text("Text")
-            Text("more text")
+            Text(
+              text = "more text",
+              modifier = Modifier.semantics { contentDescription = "custom description" })
+            Row(modifier = Modifier.semantics(mergeDescendants = true) {}) {
+              Text("Merged descendant text")
+              Text(text = "more text", modifier = Modifier.semantics { contentDescription = "custom description" })
+            }
```

Comment:
nit: maybe we can add the same support in the plugin test `composeA11y` project?

Reply:
try doing it here
https://github.com/cashapp/paparazzi/blob/master/paparazzi/paparazzi-gradle-plugin/src/test/projects/compose-a11y/src/main/java/app/cash/paparazzi/plugin/test/CompositeComposable.kt#L48-L57

Reply:
Sounds good, will do!